scripts/subctl.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+source ${SCRIPTS_DIR}/lib/utils
+source ${DAPPER_SOURCE}/scripts/lib/utils
+
+file=$(readlink -f releases/target)
+read_release_file
+
+export VERSION="${release["components.submariner-operator"]}"
```

Comment:
But how would this work if the commit-id is a sha?

I guess that in some cases we will need to rebuild subctl, probably injecting the component versions?

Reply:
Yeah I thought about it but I think these reasons we need to take into consideration:
1. Each project has the best knowledge of how to build the image/binaries so it's better to use the built ones rather than rebuild here.
2. We're probably going to do at least one RC/pre tag before (though IDK about point releases) so we should have that tag to use as the base for the official release.

We could of course revisit this in the future should one of the points above needs addressing

Reply:
> Yeah I thought about it but I think these reasons we need to take into consideration:
> 
> 1. Each project has the best knowledge of how to build the image/binaries so it's better to use the built ones rather than rebuild here.

Yes, but I think part of the plan was that in this repo we would specify the sha... and we would trigger the tag on other repos for the images to be create, instead of doing it manually etc.


> 2. We're probably going to do at least one RC/pre tag before (though IDK about point releases) so we should have that tag to use as the base for the official release.
> 
> We could of course revisit this in the future should one of the points above needs addressing

Reply:
For sure, we can further adjust it in the future.
Let's merge the patch and keep improving upon it to support more cases